src/main/java/org/dormitory/autobotsoccub/user/UserUtils.java

```diff
@@ -0,0 +1,27 @@
+package org.dormitory.autobotsoccub.user;
+
+import lombok.Getter;
+import org.telegram.telegrambots.api.objects.User;
+
+import javax.inject.Singleton;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Singleton
+public class UserUtils {
```

Comment:
It seems there is no adequate reason to make this class a bean. Let it be plain old utility class with static methods.
BTW, we're not using component scan, spring beans should be defined in corresponding contexts.

Reply:
Good, now it is old utility class with static methods.